__mocks__/fs.js

```diff
@@ -6,6 +6,13 @@ exports.readFile = (file, cb) => {
   if (file.match(/bad/i)) {
     cb('Invalid File');
   } else {
-    cb(undefined, new Buffer('File Contents'));
+    cb(undefined, new Buffer('{"firstName":"Edward","lastName":"Scissorhands"}'));
   }
 };
+
+exports.writeFile = ('file', 'new string', (err) => {
```

Comment:
You shouldn't be hardcoding parameters here. The mock is supposed to be the same input and output as the original version, so you should still have three variable parameters. This is because you're checking that the flow from `index.js` > your promise based and async/await based read > `fs.readFile` is working correctly. Therefore, you also need to check that the parameters are being passed through correctly, and thus need variable parameters.